x-pack/plugins/index_management/public/application/sections/home/template_list/template_list.tsx

```diff
@@ -218,77 +218,103 @@ export const TemplateList: React.FunctionComponent<RouteComponentProps<MatchPara
     </>
   );
 
-  const renderContent = () => {
-    if (isLoading) {
-      return (
-        <SectionLoading>
+  // Track component loaded
+  useEffect(() => {
+    uiMetricService.trackMetric(METRIC_TYPE.LOADED, UIM_TEMPLATE_LIST_LOAD);
+  }, [uiMetricService]);
+
+  let content;
+
+  if (isLoading) {
+    content = (
+      <PageLoading>
+        <FormattedMessage
+          id="xpack.idxMgmt.indexTemplatesList.loadingIndexTemplatesDescription"
+          defaultMessage="Loading templates…"
+        />
+      </PageLoading>
+    );
+  } else if (error) {
+    content = (
+      <PageError
+        title={
           <FormattedMessage
-            id="xpack.idxMgmt.indexTemplatesList.loadingIndexTemplatesDescription"
-            defaultMessage="Loading templates…"
+            id="xpack.idxMgmt.indexTemplatesList.loadingIndexTemplatesErrorMessage"
+            defaultMessage="Error loading templates"
           />
-        </SectionLoading>
-      );
-    } else if (error) {
-      return (
-        <SectionError
-          title={
+        }
+        error={error}
+      />
+    );
+  } else if (!hasTemplates) {
+    content = (
+      <EuiEmptyPrompt
```

Comment:
This empty prompt didn't originally offer the user any actions, so I added those and updated the copy to mirror our other empty prompts.

Reply:
Nice! 👍